README.md

```diff
@@ -156,10 +156,11 @@ Here's what we _do_ use:
 * For further organisation, favour alphabetical ordering (grouping by type _always_ takes precedence)
 
 
-<br/>
+</br>
```

Comment:
I don't think this is valid HTML/Markdown ... https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br